### PR TITLE
Feature cart view

### PR DESCRIPTION
**Pull request by DiegoCuevas**
* Add menu-item in the order view 
* Add increase and decrease function

### CODE REVIEW

File: src/views/cart.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+              </div>
+              <span>$/. {e.price * e.quantity}</span>
+            </article>
+          );
+        })}
+
+        <article css={cssResult}>
+          <span css={{ margin: "10px" }}>Total</span>{" "}
+          <span css={{ margin: "10px" }}>
+            $/. {Object.values(cart).reduce((acc, item) => {
```
Comment (stg101) [inline src/views/cart.js]: I like that you have used a reduce here, is very compact.

File: src/views/cart.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  if (!user) return <Redirect to="login" noThrow />;
+  return (
+    <>
+      <section
+        css={{
+          margin: "0 1em",
+          flexFlow: "column"
+        }}
+      >
+        {Object.values(cart).map(e => {
```
Comment (stg101) [inline src/views/cart.js]: I think you can use a better variable name instead of `e`.

Comment (DiegoCuevas) [reply src/views/cart.js]: I already did that, dude :)

Comment (stg101) [reply src/views/cart.js]: Nice

**Pull request merged**